core/env/assoc_spec.rb

```diff
@@ -1,23 +1,29 @@
-require_relative '../../spec_helper'
```

Comment:
Could you keep the `require_relative '../../spec_helper'` here?
It's convention to keep it as first line in specs.

Reply:
Putting the require_relative at line 1 did not work for me.  The spec_helper launches the testing, right?  If it's cited at line 1, nothing below gets interpreted before the test runs.

Reply:
It works for `core/array/join_spec.rb` for instance:
`$ ruby core/array/join_spec.rb`
The recommended way to run specs is to use `mspec` explicitly though:
`$ mspec core/array/join_spec.rb`